### PR TITLE
Monero Means Money 'en' file added

### PR DESCRIPTION
**Pull request by lh1008**
Hey everyone,

Adding 'Monero Means Money' (https://www.monerooutreach.org/news/monero-means-money.html) English version.

It should be ready to merge.

Thank you :).

Pinging @erciccione

### CODE REVIEW

File: monero-outreach-docs/en/archive/monero_means_money_en.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Monero Means Money Movie Removed From #1 List
+
+**“The Top Movie in America Last Weekend Was about privacy...briefly.” - 21th of April, 2020**
+
+Amid warnings on government privacy intrusions during the COVID-19 pandemic, the number-one movie in America — if only briefly — was about privacy. It temporarily had the sales to be #1 as its weekend sales exceeded the entire weekend (4/10-4/12) box office of other movies. It was listed as #1 for a couple hours and then was promptly removed from the movie industry data website [The Numbers](https://web.archive.org/web/20200413125904/https://www.the-numbers.com/weekend-box-office-chart).
```
Comment (erciccione) [inline monero-outreach-docs/en/archive/monero_means_money_en.md:5]: The final part of this text seem wrong. There is a dot missing and there are two URLs in one.

**Review by erciccione — CHANGES_REQUESTED** [review]

File: monero-outreach-docs/en/archive/monero_means_money_en.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/en/archive/monero_means_money_en.md:5]: Hey Erciccione, where is the dot missing? Check the link, that's how it displays the Weekend domestic Chart for April 10, 2020.

Comment (erciccione) [reply monero-outreach-docs/en/archive/monero_means_money_en.md:5]: Yes you are right. I didn't realize it was an archive.org link.

**Review by erciccione — APPROVED** [review]

File: monero-outreach-docs/en/archive/monero_means_money_en.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/en/archive/monero_means_money_en.md:5]: It's okay :)

**Pull request merged**